From 4b2fb9adb3cd46cf6fe9b7e093d3f513a44f8e14 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <fsumsal@redhat.com>
Date: Thu, 14 Dec 2023 10:54:38 +0100
Subject: [PATCH] Revert "man: mention System Administrator's Guide in
 systemctl manpage"

This reverts commit 5b2c931fb85d79db5a369a46eaeaf4ba297cbeef.

Related: RHEL-19436

rhel-only
---
 man/systemctl.xml | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/man/systemctl.xml b/man/systemctl.xml
index b73d4ac048..55310c974e 100644
--- a/man/systemctl.xml
+++ b/man/systemctl.xml
@@ -2518,17 +2518,6 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
     <xi:include href="common-variables.xml" xpointer="urlify"/>
   </refsect1>
 
-  <refsect1>
-    <title>Examples</title>
-    <para>
-            For examples how to use systemctl in comparsion
-            with old service and chkconfig command please see:
-            <ulink url="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_basic_system_settings/managing-system-services-with-systemctl_configuring-basic-system-settings">
-                    Managing System Services
-            </ulink>
-    </para>
-  </refsect1>
-
   <refsect1>
     <title>See Also</title>
     <para>
